aten/src/ATen/native/cpu/FlashAttentionKernel.cpp

```diff
@@ -654,7 +654,11 @@ void cpu_flash_attention_backward(
               fill_stub(row_ptr + last_col + 1, static_cast<accum_t>(0), kvBlockSize - last_col - 1);
             }
           }
+#ifdef _MSC_VER
           if (is_reduced_type) {
```

Comment:
I think we have a CONSTEXPR IF NOT MSVC macro somewhere for this.

Reply:
I was thinking of CONSTEXPR_EXCEPT_WIN_CUDA, but I wonder if have another macro for this.

Reply:
It should be a MSVC2019 bug, however, there is only CONSTEXPR_EXCEPT_WIN_CUDA in c10/macros/Macros.h but this is not a CUDA file.